styles/website.css

```diff
@@ -1713,28 +1713,32 @@ img {
         display: none;
     }
 }
-.infobox {
+.infobox,.warningbox {
```

Comment:
I believe in this case it's better to use `.box.info`, i.e. two classes, to clearly indicate their shared layout.

Reply:
This will mean that in-line it will have to be specified as `<div class="info box"><p>` and `<div class="warning box"><p>` with the space between the words, but I agree it makes it clearer.

Example css:
```css
.box {
    display: inline-flex;
    padding: .25em .75em .25em 0;
    border-left: .4em solid!important;
    width: 100%
}
.box::before {
    content: "";
    font: 14px/1 FontAwesome;
    font-size: 1.75em;
    margin: 0 .5em 0 .75em;
    align-self: center
}
.box.info {
    border: .1em dashed;
    border-radius: .2em;
    opacity: 80%;
    font-size: .95em
}
.box.warning {
    border: .2em outset;
    border-radius: .5em; /* this can be changed */
    box-shadow: 0 0 .2em rgba(0,0,0,.8)
}
.box.warning::before {
    content: ""
}
```